components/frontend/src/source/SourceEntityAttribute.js

```diff
@@ -12,7 +13,8 @@ export function SourceEntityAttribute({ entity, entity_attribute }) {
         cellContents && type === "datetime" ? <TimeAgoWithDate dateFirst date={cellContents} /> : cellContents
     cellContents =
         cellContents && type === "date" ? <TimeAgoWithDate dateFirst noTime date={cellContents} /> : cellContents
-    cellContents = cellContents && type.includes("integer") ? Math.round(cellContents).toString() : cellContents
+    cellContents =
+        cellContents && type.includes("integer") ? formatMetricValue("count", Math.round(cellContents)) : cellContents
```

Comment:
Is "integer" the only case, or does that also include floats? Are version numbers always labeled as type string?

Reply:
Entity attribute types can also be floats, but so far they apparently haven't needed special formatting. I've added a reference to the data model where these types are defined.